### DISCUSSION TITLE
Script problem

### DISCUSSION THREAD

**Discussion by tjzzhang** [bnpr/Malt#420] [Q&A] [answered]
When I load pre version example - basic.mesh.glsl for Shader source in the latest version, it shows the problem:
Can't open include file "Pipelines/NRP_Pipeline.glsl"
--------------------------------------------------------------------
basic.mesh.glsl--->
//Copyright (c) 2020 BlenderNPR and contributors. MIT license.

#include "Pipelines/NPR_Pipeline.glsl"

uniform vec3 ambient_color = vec3(0.1,0.1,0.1);
uniform vec4 diffuse_color = vec4(1.0,0.1,0.1,0.5);
uniform vec3 specular_color = vec3(1.0,1.0,1.0);
uniform float roughness = 0.5;

uniform sampler2D alpha_texture;

void COMMON_PIXEL_SHADER(Surface S, inout PixelOutput PO)
{
    vec3 diffuse = diffuse_color.rgb * get_diffuse_half();
    vec3 specular = specular_color * get_specular(roughness);

    vec3 color = ambient_color + diffuse + specular;
    float alpha = diffuse_color.a + luma(specular);
    alpha = clamp(alpha, 0, 1);

    PO.color.rgb = color;
    PO.color.a = alpha;
    PO.transparency_shadow_color = diffuse_color;
}

**Comment by pragma37** [comment, marked as answer]
See my answer here: https://github.com/bnpr/Malt/issues/300

These are the actual getting started docs:
https://malt3d.com/documentation/getting%20started/
